src/pages/IndividualReport.vue

```diff
@@ -0,0 +1,410 @@
+<template>
+  <div v-if="!studentData" class="flex flex-column justify-content-center align-items-center loading-container">
+    <AppSpinner style="margin-bottom: 1rem" />
+    <span>Loading Your Individual Roar Score Report...</span>
+  </div>
+
+  <div
+    v-else
+    id="individual-report-export"
+    ref="IndividualReportDownload"
```

Comment:
Style nitpick: recommend camel case with lowercase first character
```suggestion
    ref="individualReportDownload"
```